src/views/UserList.vue

```diff
@@ -1,110 +1,176 @@
 <template>
   <div>
     <NavList />
-    <br>
-    <br>
-    <br>
-    <br>
-    <br>
-    <div>
+
+    <div class="buttons">
       <a class="Cadastro">Cadastro</a>
       <a class="Listagem">Listagem</a>
-      <br>
-      <br>
-      <br>
       <button class="btn">Cadastrar Novo Usuario</button>
     </div>
     <div class="user">
-      <br>
       <br>
       <hr>
-     <a id="cpf">CPF</a>
+      <a id="cpf">CPF</a>
       <a id="FullName">Nome Completo</a>
-      <br>  
-      <br>
       <hr>
     </div>
-    <div>
+    <div id="renderResult">
       <br>
-      <div id="pagination">
-      <Pagination/>
-      <br>
-      <br>
-      </div>
-      </div>
-    </div>
+      <ul v-for="item in pageOfItems" :key="item.id">
     
+          <li id="content-cpf" >{{ item.cpf | VMask(mask)}} <br></li>
+        
+          <br>
+          <br>
+          <li id="content-name" >{{ item.fullname }}  </li>
+          <li><i @click="ModalOpen(item)" class="fa fa-eye" id="icon" aria-hidden="true"></i> </li>
+      </ul>
+      <ModalEye :User_Prop="User_Prop" v-show="openModal" />
+    </div>
+    <hr>
+    <div class="card-footer pb-0 pt-3">
+      <jw-pagination :disableDefaultStyles="true" value="Continuar" :labels="customLabels" :items="users"
+        @changePage="onChangePage"></jw-pagination>
+    </div>
+    <br>
+  </div>
 </template>
 
 <script>
-import NavList from "../components/NavList.vue"
 import axios from 'axios'
-import Pagination from '../components/Pagination.vue'
-export default {  
+import ModalEye from "@/components/ModalEye.vue";
+import NavList from "../components/NavList.vue"
+
+const customLabels = {
+  first: '<<',
+  last: '>>',
+  previous: '<',
+  next: '>'
+};
+export default {
   components: {
     NavList,
-    Pagination
-   },
-  
+  ModalEye
+
+  },
+  data() {
+    return {
+      users: [{}],
+      User_Prop: {} ,
+      pageOfItems: [],
+      customLabels,
+      mask: "###.###.###.-##",
+      openModal: false,
+      
+    };
+  },
+  created() {
+    axios.get("/api/users/")
+      .then((res) => this.users = res.data.users)
```

Comment:
O ideal seria usar um try/catch do que usar .then e .catch
```suggestion
    try {
      const response = axios.get("/api/users/")
      this.users = response.data.users
    } catch(error) {
      Alert(`Ocorreu um erro ao buscar usuários: ${error.message}`)
    }
```